qualtran/bloqs/mcmt/multi_control_multi_target_pauli.py

```diff
@@ -372,3 +373,59 @@ def build_composite_bloq(
 
     def pretty_name(self) -> str:
         return f'C^{len(self.cvs)}-NOT'
+
+
+@frozen
+class Copy(Bloq):
```

Comment:
Can we call it `Xor` and place it in `qualtran/arithmetic/bitwise` ? I would want to avoid a bloq called `Copy` since one of the very common mistakes beginners make is to assume we can "copy" data for free and do not handle the entanglement properly.